server/models/firebase/auth.js

```diff
@@ -0,0 +1,7 @@
+const firebase = require("./db");
+
+exports.createUserWithEmailAndPassword = function(email, password) {
```

Comment:
Please create a separate service to work with auth(for example server/services/AuthService.js)